1908-01-20.xml

```diff
@@ -0,0 +1,1344 @@
+                            life, to their moral welfare and to their evolu- tion in the path of progress. He hoped that
+                            in imbibing what was best in European civilization, they would avoid what was base and degrading. Great and important reforms were urgently needed in education, sanitation, land tenure, liceusing, lunaey, the admini-tration of justice and in local, municipal provincial, and national government. The discussion of these reforms the speaker reserved to the House of Commons.
+                            It was an infinite pleasure to him to find the new spirit of civic and national patriotism steadily rising in their midst. Without this spirit all their financial progress was as sand in the desert, barren and without beauty. Man does not live by bread alone, and the people who are content with rational prosperity alone are of all people the most miserable and the moat to be pitied on the face of the earth. History was strewn with the wrecks of empires and nations, which had no bigger ideal
+                            With regard to their patriotic aims therefore he was in hearty sympathy. He trusted all political parties alike would pursue constitutional and reasoned methods in advancing their just cause. Angry and revolutionary tactics bad a kvack of recoiling upon those who indulged in them, and what was still more un fortunate in retarding the wheels of progress. By subduing racial religious jealousy, by reforming social customs restrictive of liberty and justice, by affording equal opportunity to all, by personal and national sacrifice alone would they emancipate their country. England's mission in Egypt, as far as he could see, was twofold. First to enable Egyptians (and by Egyptians he meant all who adopted Egyptian nationality) to steer their own ship, and secondly, when they were managing their own country
+                        </p>
```

Comment:
you don't need to break the paragraph and division when the column breaks. Just put the `<cb>` tag right after the word "country" and before the world "to"